pkg/limayaml/default.yaml

```diff
@@ -50,7 +50,9 @@ mounts:
   sshfs:
     # Enabling the SSHFS cache will increase performance of the mounted filesystem, at
     # the cost of potentially not reflecting changes made on the host in a timely manner.
-    # Default: false
+    # Warning: It looks like PHP filesystem access does not work correctly when
```

Comment:
> PHP filesystem

All PHP apps, or just PHP Composer?

Reply:
The failing samples from https://github.com/abiosoft/colima/issues/129 seem to be just filesystem iterators, but do not involve composer. If I read the comments correctly, then the code worked up to lima 0.8.0, but has been broken in 0.8.1. On reporter claims to have verified this on 3 different machines.

Personally I've only experimented with composer. It fails because it doesn't see any of the unzipped files. However, the files are not there when composer ends, so I assumed they never got written. But maybe composer deletes the directories again?

I'm not the best person to investigate this; I've never written a PHP program in my life, and this is all nested layers of packaged things, afaict. All manual attempts to unzip things seem to have worked fine for me, as has using `git clone` instead of `unzip` to install dependencies (but it takes forever).

The fact that it affects user programs and not just composer is what makes me want to revert the default.